### PR TITLE
Update js dependencies

### PR DESCRIPTION
**Pull request by alexander-schranz**
| Q | A
| --- | ---
| Bug fix? | no
| New feature? | yes
| BC breaks? | no
| Deprecations? | no
| Fixed tickets | -
| Related issues/PRs | Needs: https://github.com/sulu/sulu/pull/4535
| License | MIT
| Documentation PR | -

#### What's in this PR?

Update js dependencies.

#### Why?

Fixing security vulnerabilities in build.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -22,32 +22,32 @@
...
+        "file-loader": "^3.0.1",
         "glob": "^7.1.2",
-        "mini-css-extract-plugin": "^0.4.0",
-        "postcss-calc": "^6.0.1",
+        "mini-css-extract-plugin": "^0.6.0",
+        "postcss-calc": "^7.0.1",
         "postcss-hexrgba": "^1.0.0",
-        "postcss-import": "^10.0.0",
+        "postcss-import": "^12.0.1",
         "postcss-loader": "^2.0.6",
```
Comment (danrot) [inline package.json:42]: There would also be a new major release, doesn't that work? In general, how did you choose which packages to update?

Comment (alexander-schranz) [reply package.json:42]: Could not get postcss-loader upgrade and css-loader upgrade to work and some others need a babel update.

**Comment by danrot** [comment]
Build is failing

**Comment by alexander-schranz** [comment]
@danrot needs sulu/sulu#4535 to get merged first. Will then rebase and build this branch.

**Pull request merged**